anesthetic/samples.py

```diff
@@ -118,30 +118,34 @@ def plot(self, ax, paramname_x, paramname_y=None, *args, **kwargs):
 
         if do_1d_plot:
             xmin, xmax = self._limits(paramname_x)
-            if plot_type == 'kde' or plot_type is None:
+            if plot_type == 'kde':
                 plot = plot_1d
             elif plot_type == 'hist':
                 plot = hist_1d
+            elif plot_type is None:
+                ax.plot([], [])
```

Comment:
Does this take care of the colours for both hist and kde, or do you need a case for each like in the 2d code below

Reply:
No, hist and plot share the same colour cycle, so it must be only one call here. Consequently the `x=[]` method would have more chance to work in the 1D case. However, also the diagonal plots didn't quite work before as issue #31 shows.

Reply:
That does make sense. I think this is too fiddly to fix the way that I was trying to do it in #21. This fix seems sensible enough, and is not any more lines. If it gets nasty further down the line then we can consider updating.